policy.go

```diff
@@ -55,6 +55,9 @@ type Policy interface {
 	// GetConditions returns the policies conditions.
 	GetConditions() Conditions
 
+	// GetMeta returns the policies arbitrary metadata set by the user.
+	GetMeta() []byte
+
```

Comment:
Add `UnmarshalMeta` here too?

Reply:
I considered this, but decided against it because I thought that `UnmarshalMeta` should stay a convenience method of the `DefaultPolicy` implementation. Similiar to `UnmarshalJSON` which is part of the default implementation but not of the interface.

Reply:
Yeah, that makes sense!